### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kuroshhashemi**
- Added Foursquare (Destination) and RoadGoat (POI) and Viator (tours)
- Reorganized because many of the API's listed in other places are in fact POI (Point of Interest) API's which belong together
- GetYourGuide is tours along with Viator

### CODE REVIEW

**Comment by kuroshhashemi** [comment]
Should also be linked to https://github.com/unseen1980/awesome-travel/issues/3

File: README.md [hunk 1]
```diff
@@ -76,31 +76,41 @@ A curated list of awesome travel resources to help you build the next travel app
...
 | Olery | Olery Review Data API | [Go!](http://www.olery.com/api/) |
 | Yelp | Yelp Fusion API | [Go!](https://www.yelp.com/developers/documentation/v3) |
+| Amadeus | Points of Interest API | [Go!](https://developers.amadeus.com/self-service/category/destination-content/api-doc/points-of-interest)
+| Minube | Minube search API | [Go!](http://papi.minube.com/docs/) |
+| Foursquare | Foursquare Places API | [Go!](https://foursquare.com/) |
+| Sygic | Sygic Travel API | [Go!](http://docs.sygictravelapi.com/) |
+| Triposo | Travel Content API | [Go!](https://www.triposo.com/api/) |
 
-### Travel Planning
+### Itenirary
```
Comment (unseen1980) [inline README.md:100]: Small typo here ```Itinerary```

File: README.md [hunk 2]
```diff
@@ -76,31 +76,41 @@ A curated list of awesome travel resources to help you build the next travel app
 | Gadventures | Adventure Activities | [Go!](https://developers.gadventures.com/docs/index.html) |
 | PredictHQ | Events API for Travel | [Go!](https://www.predicthq.com/events/travel) |
 
-### Reviews
+### Destinations/Cities/Countries Content
+
+| API | Description | Link |
+|---|---|---|
+| RoadGoat | Content API for 4.3M cities, towns, countries, & neighborhoods | [Go!](https://www.roadgoat.com/business/cities-api) |
```
Comment (unseen1980) [inline README.md:83]: Thanks for including RoadGoat 😉

**Comment by unseen1980** [comment]
Please check my small comment.  Happy to merge after that. Thanks 👍

File: README.md (hunk 1 above)
Comment (kuroshhashemi) [reply README.md]: @unseen1980 Nice catch! Fixed in latest commit

**Comment by unseen1980** [comment]
Thanks!

**Pull request merged**